CondTools/SiStrip/test/SiStripFedCablingBuilder_cfg.py

```diff
@@ -3,10 +3,9 @@
 process = cms.Process("FedCablingBuilder")
 
 process.MessageLogger = cms.Service("MessageLogger",
+    threshold = cms.untracked.string('INFO'),
     debugModules = cms.untracked.vstring(''),
-    cablingBuilder = cms.untracked.PSet(
-        threshold = cms.untracked.string('INFO')
-    ),
+    #destinations = cms.untracked.vstring('cout'),
```

Comment:
should this be removed?

Reply:
one might want to redirect to cout when not running the unit test.
I am dismissing the comment.